core/src/poh_recorder.rs

```diff
@@ -500,6 +525,7 @@ impl PohRecorder {
                 ("prepare_send_us", self.prepare_send_us, i64),
                 ("send_us", self.send_us, i64),
                 ("ticks_from_record", self.ticks_from_record, i64),
+                ("total_sleep_us", self.total_sleep_us, i64),
```

Comment:
heh, unrelated to this PR,  but it seems like `report_metrics()` is only reported on `record()`, which means we won't get data from non leaders? Do we want to report on `tick()` as well?

Reply:
so in tick(), will bank_clone.slot() give me the accurate slot after a tick? slot is an input to report.

Reply:
ah never mind, I think I see what it's trying to do. You're right, the purpose is to aggregate all the ticks, including virtual ticks into the metrics for the next slot. The non-specific slot version is done in `report_service::report`.